Dockerfile.template

```diff
@@ -0,0 +1,5 @@
+FROM docker.elastic.co/elasticsearch/elasticsearch:{version}.0-SNAPSHOT
```

Comment:
Why use `0-SNAPSHOT` at all? `x.y.0` should always exist, maybe with the exception of prerelease versions.